Orange/widgets/regression/owlinearregression.py

```diff
@@ -171,6 +173,27 @@ def commit(self):
         self.send("Coefficients", coef_table)
 
     def send_report(self):
+        self.report_items((("Name", self.learner_name),))
+
+        items = OrderedDict()
+
+        if self.reg_type == OWLinearRegression.OLS:
+            items["Regularization"] = "No Regularization"
+        elif self.reg_type == OWLinearRegression.Ridge:
+            items["Regularization"] = "Ridge Regression (L2)"
+            items["Regularisation Strength"] = "{}".format(self.alphas[self.alpha_index])
```

Comment:
Regularisation -> Regularization (just for consistency with above).

Better still, merge the two lines: write "Ridge regression (L2) with α=0.5". Do this for ridge and lasso. For elastic, "Elastic net regression with α=0.5 and L1:L2 ration of 0.2:0.8".